packages/core/lib/createConfig.js

```diff
@@ -0,0 +1,53 @@
+const createFileTypesMapping = require('./createFileTypesMapping')
+const defaultFileTypes = require('./defaultFileTypes')
+const {assertConfigSetters, invokeConfigSetters} = require('./configSetters')
+const {invokePreHooks, invokePostHooks} = require('./hooks')
```

Comment:
We used to have spaces inside {}, not sure if there are any linting rules for that.

Reply:
`standard` didn't catch these, I'll fix manually then.

Reply:
I’m trying to sell Prettier to @andywer — but after 1.0 final release ;-)